### PR TITLE
[Mellanox]support led for fan/psu and fan's direction

### PR DESCRIPTION
**Pull request by stephenxs**
**- What I did**
support led for fan/psu and fan direction
It depends on [add fan direction "not applicable" #67](https://github.com/Azure/sonic-platform-common/pull/67)
**- How I did it**
Fetch that information from sysfs.

**- How to verify it**
set led for fan/psu and then check
get fan direction
[test-led-fandir.txt](https://github.com/Azure/sonic-buildimage/files/3873991/test-led-fandir.txt)

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: platform/mellanox/mlnx-platform-api/sonic_platform/fan.py [hunk 1]
```diff
@@ -15,23 +15,28 @@
...
-LED_ON = 1
-LED_OFF = 0
+LED_ON = '1'
+LED_OFF = '0'
 
 PWM_MAX = 255
 
 FAN_PATH = "/var/run/hw-management/thermal/"
 LED_PATH = "/var/run/hw-management/led/"
+# fan_dir only exist Spectrum 2 switches
```
Comment (liat-grozovik) [inline platform/mellanox/mlnx-platform-api/sonic_platform/fan.py]: Please change the comment that it is currently not supported in spectrum 1 and later new Silicon should not be addressed and will have it supported by default

Comment (stephenxs) [reply platform/mellanox/mlnx-platform-api/sonic_platform/fan.py]: fixed.

**Comment by stephenxs** [comment]
retest this, please

**Comment by stephenxs** [comment]
retest this, please

**Comment by stephenxs** [comment]
retest this, please

**Comment by jleveque** [comment]
Retest vsimage please

**Pull request merged**